lib/api_client/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "api_client"
```

Comment:
How about changing this to `vector-api-client`?

1. Uses underscores - this notation is kind of more wide-spread in the crates as I see, and it has a special-cased support in cargo.
2. Name is scoped with `vector` - we're unlocking potential publishing of this crate to crates.io without extra renaming steps.

Reply:
👍 done in 7f7c7dde5319da2417c47f2169fd341f15dcfdf9

Reply:
Sweet! We might also rename the dir accordingly, but that's not as important.

Reply:
Ah, nvm diffs didn't update properly!

Reply:
@MOZGIII - the folder was renamed to `lib/vector-api-client` -- do you mean something else?